src/features/landing/components/Footer/FooterHead/FooterHead.tsx

```diff
@@ -0,0 +1,50 @@
+import { Container } from '@mui/material';
+
+import CustomLink from '@/components/ui/CustomLink/CustomLink';
+
+import VilnaMainIcon from '../../Icons/VilnaMainIcon/VilnaMainIcon';
+import FooterConfidential from '../FooterConfidential/FooterConfidential';
+
+const styles = {
+  mainContainer: {
+    width: '100%',
+    maxWidth: '1192px',
+    margin: '0 auto',
+    padding: '11px 0px 7px 0px',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  logo: {
+    width: '130px',
+    justifySelf: 'flex-start',
+    textDecoration: 'none',
+    color: 'black',
+  },
+  confidentialsBox: {
+    display: 'flex',
+    gap: '8px',
+  },
+  confidentialLink: {
+    borderRadius: '8px',
+    background: '#F4F5F6',
+    color: '#969B9D',
+    fontFamily: 'Inter-Regular, sans-serif',
+    fontSize: '16px',
+    fontStyle: 'normal',
+    fontWeight: '500',
+    lineHeight: '18px',
+    padding: '8px 16px',
+  },
+};
+
+export default function FooterHead() {
+  return (
+    <Container sx={{ ...styles.mainContainer }}>
```

Comment:
why you use Container component? Grid component will be better for this reason, read about Container and Grid components and what possibilities they have
https://mui.com/material-ui/api/container/
https://mui.com/material-ui/react-grid/